profiles/arch/riscv/package.use.mask

```diff
@@ -1,6 +1,14 @@
 # Copyright 2019-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Ye Cao <ye.c@rioslab.org> (2021-06-30)
+# # Dependencies not keyworded, not tested
+sci-libs/fftw mpi
+media-sound/alsa-utils libsamplerate
+dev-python/ipython doc matplotlib nbconvert notebook
```

Comment:
Why do you mask _doc_ and _matplotlib_? Your PR takes care of all the missing keywords on the dependencies pulled in by these two.

Reply:
Thank you! I've fixed the issue just now.

Reply:
Thank you! fixed it.